lascli/actions/workflows.py

```diff
@@ -0,0 +1,262 @@
+                'jsRemoteComponent': parameters['remote_component_asset_id'],
+                'fieldConfig': parameters['field_config_asset_id']
+            }
+        },
+        name=parameters['name'],
+        description=create_tag,
+    )
+
+    return preprocess['transitionId'], postprocess['transitionId'], manual['transitionId']
+
+
+def create_default_workflow(las_client: Client, name: str, **optional_args):
+    timestamp = datetime.now()
+    create_tag = f'Created by CLI at {timestamp.isoformat()} (tag:{uuid.uuid4().hex})'
+    
+    if model_id := optional_args.get('from_model_id'):
+        try:
+            docker_secret_id, cradl_secret_id = create_secrets(
+                las_client=las_client,
+                create_tag=create_tag,
+                username=optional_args.get('username'),
+                password=optional_args.get('password'),
+            )
+
+            component_asset_id, field_config_id = create_assets(
+                las_client=las_client,
+                model_id=model_id,
+                create_tag=create_tag,
+            )
+            
+            dataset_id = create_dataset(
+                las_client=las_client,
+                name=f'Dataset for {name}',
+                create_tag=create_tag,
+            )
+
+            preprocess_id, postprocess_id, manual_id = create_transitions(
+                las_client=las_client,
+                cradl_secret_id=cradl_secret_id,
+                parameters={
+                    'remote_component_asset_id': component_asset_id,
+                    'field_config_asset_id': field_config_id,
+                    'secret_id': docker_secret_id,
+                    'preprocess_image': optional_args['preprocess_image'],
+                    'postprocess_image': optional_args['postprocess_image'],
+                    'name': name,
+                    'model_id': model_id,
+                    'dataset_id': dataset_id,
+                },
+                create_tag=create_tag,
+                created_ids=created_ids,
+            )
+
+            spec = create_workflow_spec(
+                model_id=model_id,
+                preprocess_transition_id=preprocess_id,
+                postprocess_transition_id=postprocess_id,
+                manual_transition_id=manual_id,
+                tag=create_tag,
+            )
+            
+            workflow = las_client.create_workflow(
+                specification=spec,
+                name=name,
+                description=create_tag,
+            )
+
+            print(f'Created workflow {workflow["workflowId"]}')
+
+            return workflow
+        except Exception as e:
+            traceback.print_exc()
+
+            print('Cleaning up resources ...')
+            
+            def for_each(fn, resources, msg=None):
+                for resource in filter(lambda r: r, resources or []):
+                    print(f'{msg} ({resource}) ...')
+                    fn(resource)
```

Comment:
Do we want the entire clean-up phase to stop if any of the clean-up steps fail or should we add some exception handling here too?